procedures/katello/branch.md.erb

```diff
@@ -49,6 +49,7 @@
   - [ ] [pulpcore](https://github.com/theforeman/puppet-pulpcore)
   - [ ] [foreman_proxy_content](https://github.com/theforeman/puppet-foreman_proxy_content)
   - [ ] [katello](https://github.com/theforeman/puppet-katello)
+- [ ] Commit the [settings file](https://github.com/theforeman/theforeman-rel-eng/blob/master/releases/katello/<%= release %>/settings) to the `theforeman-rel-eng` repository
```

Comment:
Would it make sense to add a note about how to create it? or link to a commit that adds it for a previous version?